### ISSUE TITLE
Conda install: one_neuron.py example dumps core.

### ISSUE THREAD

**Issue by brenthuisman** [nest/nest-simulator#1728]
**Describe the bug**
I've installed Nest through conda: `conda install -c conda-forge nest-simulator`.

I've taken a simple example from the docs: <https://nest-simulator.readthedocs.io/en/latest/auto_examples/one_neuron.html>. Running this example gives the following output:

```
[~/code/testjes] python one_neuron.py 
[INFO] [2020.8.21 11:20:5 /home/conda/feedstock_root/build_artifacts/nest-simulator_1583214388159/work/nestkernel/rng_manager.cpp:217 @ Network::create_rngs_] : Creating default RNGs
[INFO] [2020.8.21 11:20:5 /home/conda/feedstock_root/build_artifacts/nest-simulator_1583214388159/work/nestkernel/rng_manager.cpp:260 @ Network::create_grng_] : Creating new default global RNG
python: /home/conda/feedstock_root/build_artifacts/nest-simulator_1583214388159/work/sli/scanner.cc:581: bool Scanner::operator()(Token&): Assertion `in->good()' failed.
Afgebroken (geheugendump gemaakt)
```

This is in an otherwise clean conda environment. The error doesn't tell me enough to troubleshoot myself, so I post it here.

**Desktop/Environment (please complete the following information):**
 - Ubuntu 20.04
 - bash (default)
 - Python 3.7 through Conda
 - Nest v2.20
 - Conda

**Comment by anandtrex** [comment]
I'm having a similar issue. Any update on why this might be happening?

**Comment by steffengraber** [comment]
@brenthuisman Does your issue still exist? As far as I can see you are using `NEST v2.20.0` with the `latest` version of the `one-neuron`-example. Could you please check with this example:

   https://github.com/nest/nest-simulator/blob/v2.20.0/pynest/examples/one_neuron.py

@anandtrex Could you please describe your environment?

**Comment by brenthuisman** [comment]
@steffengraber Meanwhile I've switched to getting Nest from the neuroscience repository, so I don't know the status of the issue.

**Comment by anandtrex** [comment]
@steffengraber My problem was solved after updating to nest-2.20.1. (I was running single-threaded nest combined with python multiprocess).

**Comment by steffengraber** [comment]
The issue seems to be solved for everyone. Therefore I close it.
